### PR TITLE
Add ability to treat rule violations as errors

### LINKED ISSUE #522: Enable marking individual static analyzer rules as error (instead of warning)
I investigated the effect of TreatwarningsAsErrors - it has no effect on .sqlproj - the only way to make a analyzer problem an error is to mark it explicitly:

```xml
<RunSqlCodeAnalysis>True</RunSqlCodeAnalysis> 
<SqlCodeAnalysisRules>+!Microsoft.Rules.Data.SR0001;+!Microsoft.Rules.Data.SR0008</SqlCodeAnalysisRules>
```

### PR DESCRIPTION
**Pull request by ErikEJ**
fixes #522

### CODE REVIEW

**Review by jmezach — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -542,15 +542,17 @@ Static code analysis can be enabled by adding the following to the project file:
 <Project Sdk="MSBuild.Sdk.SqlProj/2.7.0">
   <PropertyGroup>
     <RunSqlCodeAnalysis>True</RunSqlCodeAnalysis>
-    <CodeAnalysisRules>-SqlServer.Rules.SRD0006;-Smells.*</CodeAnalysisRules>
+    <CodeAnalysisRules>-SqlServer.Rules.SRD0006;-Smells.*;+!SqlServer.Rules.SRN0005</CodeAnalysisRules>
```
Comment (jeffrosenberg) [inline README.md:545]: I would recommend against using the `+!` syntax in our only example. Instead, I'd keep this example the same as it was, and add a second example below showing how to force a rule violation. I think that would be clearer.

**Review by jeffrosenberg — APPROVED** [review]

**Comment by ErikEJ** [comment]
@jmezach Time to consider a 2.7.0 release? 😄

**Comment by ErikEJ** [comment]
@jmezach Just FYI, did manage to do some field testing, and a small update is coming soon based on that

**Comment by ErikEJ** [comment]
@jmezach Should this not be downloading the sdk and template packages instead of the test package?

# Download artifacts
    - name: download-artifact
      uses: actions/download-artifact@v4
      id: download-nugetpackages
      with:
        name: test-package
        path: ~/nuget-packages

**Comment by jmezach** [comment]
Yeah, that makes more sense ;).

**Comment by jmezach** [comment]
Release 2.7.0 is out there and includes your latest fixes.

**Comment by ErikEJ** [comment]
@jmezach Wonderful! Thanks

**Pull request merged**